src/components/Card.js

```diff
@@ -47,6 +78,11 @@ export default class Card {
     cardTitleEl.textContent = this._name;
     cardImageEl.src = this._link;
     cardImageEl.alt = this._name;
+    if (this._isLiked) {
+      this._cardElement
```

Comment:
Needs correction: there should not be code duplication, reuse `this.like` method here